FV3/atmos_cubed_sphere/model/fv_arrays.F90

```diff
@@ -1158,6 +1164,24 @@ module fv_arrays_mod
 
   end type fv_grid_bounds_type
 
+  type fv_coarse_grid_bounds_type
+
+     integer :: is_coarse, ie_coarse, js_coarse, je_coarse
+
+  end type fv_coarse_grid_bounds_type
+
+  type fv_coarse_graining_type
+
+     type(fv_coarse_grid_bounds_type) :: bd
+     type(domain2d) :: domain
+     integer :: factor
+     integer :: target_resolution
+     integer :: diagnostic_axes(4)
```

Comment:
These are integer IDs set up through `diag_manager_mod.diag_axis_init`, which one needs to use to register diagnostics on a given grid (in this case the coarse grid).  They define the dimensions of the variables in the netCDF files produced.  They are useful to have on hand so that we can eventually pass them to the physics, so that the coarse physics diagnostics will be written out on the same coarse dimensions as the coarse dycore diagnostics.

I've restructured this a little bit and added some comments to the code.